src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -0,0 +1,294 @@
+﻿<Window xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        x:Class="Dynamo.Wpf.Views.PreferencesWindow"
+        x:Name="DynamoPreferencesWindow"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:p="clr-namespace:Dynamo.Wpf.Properties;assembly=DynamoCoreWpf"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:views="clr-namespace:Dynamo.Wpf.Views"
+        WindowStartupLocation="CenterOwner"
+        WindowStyle="None"
+        ResizeMode="NoResize"
+        mc:Ignorable="d" 
+        Height="474" 
+        Width="530" >
```

Comment:
Can you add some comments before each UI components in this xaml for future reference? Thanks!

Reply:
@QilongTang I added extra comment in the PreferencesView.xaml. Thanks